### PR TITLE
libcephfs: cleanups

### PR DESCRIPTION
**Pull request by renhwztetecs**

### CODE REVIEW

**Comment by xiexingguo** [comment]
retest this please

**Comment by xiexingguo** [comment]
retest this please

File: src/test/libcephfs/test.cc [hunk 1]
```diff
@@ -999,7 +999,13 @@ TEST(LibCephFS, BadFileDesc) {
 
   ASSERT_EQ(ceph_get_file_stripe_unit(cmount, -1), -EBADF);
   ASSERT_EQ(ceph_get_file_pool(cmount, -1), -EBADF);
+  char poolname[80];
+  ASSERT_EQ(ceph_get_file_pool_name(cmount, -1, poolname, sizeof(poolname));
```
Comment (jcsp) [inline src/test/libcephfs/test.cc]: missing second argument to ASSERT_EQ, does not compile

Comment (renhwztetecs) [inline src/test/libcephfs/test.cc]: thanks，I fix it

**Pull request merged**